src/TailwindCssPreset.php

```diff
@@ -29,17 +28,20 @@ public static function installAuth()
     protected static function updatePackageArray(array $packages)
     {
         return array_merge([
-            '@tailwindcss/ui' => '^0.3',
-            'autoprefixer' => '^9.6',
-            'laravel-mix' => '^5.0.1',
-            'postcss-import' => '^12.0',
-            'postcss-nested' => '^4.2',
-            'tailwindcss' => '^1.8',
-            'vue-template-compiler' => '^2.6.11',
+            '@tailwindcss/forms' => '^0.2.1',
+            'autoprefixer' => '^10.2.3',
+            'laravel-mix' => '^6.0.6',
+            'postcss' => '^8.2.4',
+            'resolve-url-loader' => '^3.1.2',
+            'tailwindcss' => '^2.0.2',
+            'vue' => '^2.6.12',
+            'vue-loader' => '^15.9.6',
+            'vue-template-compiler' => '^2.6.12',
```

Comment:
Vue was only in this array because Laravel used to ship with it by default. It can be dropped entirely now, seeing as it's not a requirement of Tailwind and is no longer shipped with Laravel.